grafana_objects/testdata/createupdate.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Please add directory "fixture" under the directory "testdata", and place the files there.

Reply:
done